HyperV/scripts/create-environment.ps1

```diff
@@ -273,16 +264,21 @@ ExecRetry {
         Get-ChildItem $buildDir\compute-hyperv
     }
     pushd $buildDir\compute-hyperv
-    & pip install -e $buildDir\compute-hyperv
+    if (($branchName -eq 'stable/liberty') -or ($branchName -eq 'stable/mitaka')) {
+        & pip install -c $buildDir\requirements\upper-constraints.txt -U .
+    }
+    else {
+        & pip install -e $buildDir\compute-hyperv
+    }
     if ($LastExitCode) { Throw "Failed to install compute-hyperv from repo" }
     popd
 }
 
 $novaConfig = (gc "$templateDir\nova.conf").replace('[DEVSTACK_IP]', "$devstackIP").Replace('[LOGDIR]', "$openstackLogs").Replace('[RABBITUSER]', $rabbitUser)
 $neutronConfig = (gc "$templateDir\neutron_hyperv_agent.conf").replace('[DEVSTACK_IP]', "$devstackIP").Replace('[LOGDIR]', "$openstackLogs").Replace('[RABBITUSER]', $rabbitUser)
 
-if (!$branchName.CompareTo('master')){
-    $novaConfig = $novaConfig.replace('hyperv.driver.HyperVDriver', 'compute_hyperv.driver.HyperVDriver')
+if (($branchName -eq 'stable/liberty') -or ($branchName -eq 'stable/mitaka')) {
+    $novaConfig = $novaConfig.replace('compute_hyperv.driver.HyperVDrvier', 'hyperv.nova.driver.HyperVDriver')
```

Comment:
It depends which driver you want to use. The right path for stable branches (if you want to use the compute-hyperv driver) is
hyperv.nova.driver.HyperVDriver
(https://github.com/openstack/compute-hyperv/blob/master/hyperv/nova/driver.py)

Otherwise, you'll be using the in-tree Nova driver.